api-spec/STAC-extensions.yaml

```diff
@@ -640,10 +640,15 @@ components:
     sortby:
       name: sortby
       in: query
-      description: Allows sorting results by the specified properties
+      description: >
+        An array of property names, prefixed by either "+" for ascending or "-"
+        for descending. If no prefix is
+
```

Comment:
That's a bit too much space here ;-)

Reply:
That's what npm generated -- I will manually edit it to clean it up.

Reply:
Actually, if you look through STAC-extensions.yaml, it's filled with descriptions that messed up / double spacing all over.

Reply:
Oh, I see. Hmm... bummer. I opened a separate issue on this (and a related issue): #730